templates/env.j2

```diff
@@ -0,0 +1,14 @@
+DJANGO_SETTINGS_MODULE="camac.settings"
+DJANGO_DATABASE_NAME="{{ camac_db_dbname }}"
+DJANGO_DATABASE_USER="{{ camac_db_username }}"
+DJANGO_DATABASE_PASSWORD="{{ camac_db_password }}"
+DJANGO_DATABASE_HOST="{{ camac_db_host }}"
+DJANGO_DATABASE_PORT="{{ camac_db_port }}"
+DJANGO_ALLOWED_HOSTS="{{ camac_python_django_allowed_hosts }}"
+DJANGO_SECRET_KEY="{{ camac_python_django_secret_key }}"
+DJANGO_MEDIA_ROOT="{{ camac_python_django_media_root }}"
+DJANGO_DEFAULT_FROM_EMAIL="{{ camac_python_django_default_from_email }}"
+DJANGO_SERVER_EMAIL="{{ camac_python_django_server_email }}"
+DJANGO_EMAIL_HOST="{{ camac_python_django_email_host }}"
+APPLICATION_ENV="{{ camac_env }}"
```

Comment:
this is only needed for dev purposes and should not be overwritten on a deployment. Better remove it.

Reply:
Thisi leads to `django.core.exceptions.ImproperlyConfigured: Set the APPLICATION_ENV environment variable`. I will fave a look at the settings.py.

Reply:
`ENV = env.str("APPLICATION_ENV")` in the `settings.py` causes the exception.

Reply:
Probably we should just use
`ENV = env.str("APPLICATION_ENV", "production")`

Reply:
totally agree with @czosel

Reply:
Fixed it.